log-collector-script/linux/eks-log-collector.sh

```diff
@@ -526,6 +526,14 @@ get_networking_info() {
   fi
 
   cp /etc/resolv.conf "${COLLECT_DIR}"/networking/resolv.conf
+
+  # collect ethtool -S for all interfaces
+  INTERFACES=$(ip -o a | awk '{print $2}' | sort -n | uniq)
+  for ifc in ${INTERFACES}; do
+    echo "Interface ${ifc}" >> "${COLLECT_DIR}"/networking/ethtool.txt
```

Comment:
i'd suggest modifying the first line to just `>` to initialize the file:
```
echo "Interface ${ifc}" > "${COLLECT_DIR}"/networking/ethtool.txt
```

so if this script is run multiple times, you get only the output from the last run.

Reply:
ah cool. thanks!